test/check-multi-machine

```diff
@@ -87,6 +87,11 @@ class TestMultiMachine(MachineCase):
         self.machine3.wait_boot()
         self.machine3.execute("hostnamectl set-hostname machine3")
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1297745
+        if self.machine.image == "rhel-atomic":
```

Comment:
Would prefer a naughty file so we can track when this issue goas away. Is it possible to implement that way?